deno.lock

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Ooooo lockfile! This is a new one for me in a deno project. Should we have one of these now?

Reply:
I haven't spent too much time thinking about this. Right now I'm leaning towards us including it, but @jesselawson if you could remove this from this PR I think that will make approving this one easier!

Reply:
Yes I'll remove it -- wanted to keep it here to make sure it's on _someone's_ radar.

Reply:
[Removed](https://github.com/slackapi/deno-slack-sdk/pull/123/commits/09b68c60869fbf3255956ca7f012bb150a2b64b6)